data/favorites/domain.yml

```
@@ -0,0 +1,89 @@
+version: "3.1"
+
+intents:
+  - intent_favorites_show
+  - intent_favorites_add
+  - intent_favorites_delete
+
+slots:
+  favorites_service:
+    type: text
+    mappings:
+      - type: custom
+      - type: from_intent
+        value: "unsure"
+        intent: intent_core_learn_more
+        conditions:
+          - active_loop: form_favorites_add
+            requested_slot: favorites_service
+          - active_loop: form_favorites_delete
+            requested_slot: favorites_service
+  favorites_unhappy:
+    type: bool
+    mappings: &custom
+      - type: custom
+  favorites_suggestions:
+    type: list
+    mappings: *custom
+
+forms:
+  form_favorites_add:
+    required_slots:
+      - favorites_service
+
+  form_favorites_delete:
+    required_slots:
+      - favorites_service
+
+actions:
+  - action_favorites_reset
+  - action_favorites_extract
+  - validate_form_favorites_add
+  - validate_form_favorites_delete
+
+responses:
+  utter_favorites_show:
+    - text: "You can access and manage your Favorites anytime from the [homepage](/) and the Services dropdown in the top left corner."
+  utter_favorites_how:
+    - text: "You can favorite services from the [All Services](/allservices) page by clicking the star icon."
+  utter_favorites_next:
+    - text: "What would you like to do next?"
+      buttons:
+        - title: "View my other Favorites."
+          payload: "/intent_favorites_show"
+        - title: "Favorite a service."
+          payload: "/intent_favorites_add"
+        - title: "Remove a Favorite."
+          payload: "/intent_favorites_delete"
+        - title: "Something else."
+          payload: "/intent_core_other"
```

I think we can omit the dots in the buttons.

```suggestion
        - title: "View my other Favorites"
          payload: "/intent_favorites_show"
        - title: "Favorite a service"
          payload: "/intent_favorites_add"
        - title: "Remove a Favorite"
          payload: "/intent_favorites_delete"
        - title: "Something else"
          payload: "/intent_core_other"
```